Formula/c/cargo-fuzz.rb

```diff
@@ -0,0 +1,19 @@
+class CargoFuzz < Formula
+  desc "Command-line helpers for fuzzing"
+  homepage "https://rust-fuzz.github.io/book/cargo-fuzz.html"
+  url "https://github.com/rust-fuzz/cargo-fuzz/archive/refs/tags/0.12.0.tar.gz"
+  sha256 "d7c5a4589b8b5db3d49113e733553c286ed8b50800cbdb327b71a1c1f7c648f0"
+  license "Apache-2.0"
+
+  depends_on "rust" => [:build, :test]
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system "cargo", "init"
+    system "#{bin}/cargo-fuzz", "init"
+    assert_predicate testpath/"fuzz/Cargo.toml", :exist?
+  end
```

Comment:
Can you use `cargo` from `rustup-init` instead to show that this can be used with a different toolchain? See [`cargo-binutils.rb`](https://github.com/Homebrew/homebrew-core/blob/7a79cd34621614aee457a8ca9f03aef222affb0b/Formula/c/cargo-binutils.rb#L28-L65) for example.

Reply:
Thanks for the review. See if it is better now.